client/src/App.js

```diff
@@ -37,22 +53,34 @@ function App() {
   const [status, setStatus] = useState(false);
   const [newUser, setNewUser] = useState(false);
 
+  /**
+   * TODO: Get if the user is logged in
+   */
+  let loggedUser;
```

Comment:
Since we use hook, we can do this way:
put const [loggedUser,setLoggedUser]=useState(false) at the beginning of the function and we can delete   const [status, setStatus] = useState(false); Once user successfully logins we can use "setLoggedUser(true)" to change loggedUser's value.